crates/autopilot/src/run_loop.rs

```diff
@@ -268,6 +275,16 @@ impl RunLoop {
                 return;
             }
 
+            tracing::info!("saving fee policies");
+            if let Err(err) = self
+                .persistence
+                .store_fee_policies(auction_id, fee_policies)
```

Comment:
This is on the hot path (i.e. can limit our throughput by delaying the start of the next auction).
Since this information is pretty important we should probably not offload that to a background task (in case that fails) but we should optimize performance of that call by parallelizing it or preferably having a single batch insert.

Reply:
Done. I genually tried to avoid [this](https://github.com/cowprotocol/services/blob/3aaa3d91232ce7999204fd5c40c2e844c9ac45d5/crates/autopilot/src/infra/persistence/mod.rs#L117) and [this](https://github.com/cowprotocol/services/blob/3aaa3d91232ce7999204fd5c40c2e844c9ac45d5/crates/autopilot/src/infra/persistence/mod.rs#L120) by using something like `Iter<Item = IntoIter<Item = FeePolicyRow>>`, but couldn't find a proper way to make it work with async function.

Reply:
Yeah, iterators and async functions can get pretty hairy due to the references. Optimizing this to the fullest is probably not worth the headache.
One small thing that could be done is to omit the allocation of another vector for each batch with:
```rust
            database::fee_policies::insert_batch(&mut ex, chunk.iter().cloned())
```